configuration/jsonkeyvalues/vaccinationReportConfig.json

```diff
@@ -0,0 +1,11 @@
+{
+        "report.vaccination.active": "true",
+        "report.ecv.active": "true",
+        "report.prenatalVisitType": "35ba9aff-901c-49dc-8630-a59385480d18",
+        "report.vaccinations": "3c25e650-ccfa-48da-b276-a7872796b113",
+        "report.vaccinationSequenceNumber": "1f4109b3-b201-46df-a341-dcfa5299a187",
+        "report.boosterSequenceNumber": "39741bfe-36a0-4a5b-9c64-fd4caaadca01",
+        "report.vaccinationList":"9199a4c6-b15f-482c-aa0e-c839bd0cf535,a5bda1d3-281e-4421-bd73-6d649381a9a5 1,a5bda1d3-281e-4421-bd73-6d649381a9a5 2,a5bda1d3-281e-4421-bd73-6d649381a9a5 3,5d7c570a-f1f8-4e80-8804-e8d8e94ebd80,19e9c51d-e0de-4b2c-8eb0-598dbd11187c 1,19e9c51d-e0de-4b2c-8eb0-598dbd11187c 2,19e9c51d-e0de-4b2c-8eb0-598dbd11187c 3,5ba4e40a-28f9-4c21-b589-add0c9cdec78 1,5ba4e40a-28f9-4c21-b589-add0c9cdec78 2,0a28eec8-0353-415f-9eee-6f47ff67f4ce 1,0a28eec8-0353-415f-9eee-6f47ff67f4ce 2,67b20a95-eaa4-4a54-ba8c-2e3c8aa2f327 1,67b20a95-eaa4-4a54-ba8c-2e3c8aa2f327 2,67b20a95-eaa4-4a54-ba8c-2e3c8aa2f327 3,9f24e11b-24e7-4a44-917b-645a461defc3,3afaa3ce-0dbe-4ac8-964a-1913ee930d34 1,3afaa3ce-0dbe-4ac8-964a-1913ee930d34 2",
+        "report.ecvList" : "9199a4c6-b15f-482c-aa0e-c839bd0cf535,a5bda1d3-281e-4421-bd73-6d649381a9a5 2,5d7c570a-f1f8-4e80-8804-e8d8e94ebd80,19e9c51d-e0de-4b2c-8eb0-598dbd11187c 3,5ba4e40a-28f9-4c21-b589-add0c9cdec78 2,0a28eec8-0353-415f-9eee-6f47ff67f4ce 1"
```

Comment:
I have added an explanation in the readme on the PR [here](https://github.com/mekomsolutions/openmrs-module-commonreports/pull/29/commits/cd2f8c7a43bd9dfd23f8de764b6b24f7a8cfeb6b#diff-49dac642829808458caf6502613c88af5d8eecb4ce96c89ef8b8c6bdb13102c2R24-R30) otherwise the trailing number are booster/sequence numbers